test_commands.py

```diff
@@ -24,6 +31,12 @@ def testCreate(self):
         info = rts.info(4)
         self.assertEqual(20, info.retention_msecs)
         self.assertEqual('Series', info.labels['Time'])
+        if version is not None and version >= 14000:
```

Comment:
```suggestion
        if version is None or version < 14000 return
```
What do you think about something like this?